rpi/setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+# set the CPU Scaling governor to 'performance'
+echo "Setting CPU Scaling Governor to 'performance'"
+echo 'performance' > /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor
```

Comment:
We are not testing which kernel version the OS has when running this command. Does this have any effect when running on older kernel versions?

Reply:
Very good point @rahul-thakoor - but my hunch is there is no reason not to use 'performance' when we suspect the block will need it. WDYT?

Reply:
I think that makes sense. I'm only wondering if the behaviour is the same on older versions? The new kernel has a different setting as default right but the 'performance' setting will still be understood on older versions?